context/context.go

```diff
@@ -11,14 +11,15 @@ import (
 )
 
 type Context struct {
-	Deployer        deployers.Deployer
-	Vcs             services.Sources
-	Binaries        services.Binaries
-	Hooks           []hooks.Hooks
-	LockSystem      utils.Lock
-	ScreenMandatory bool
-	Log             *log.Logger
-	Metrics         *utils.MetricsRegistry
```

Comment:
:/

Reply:
?

Reply:
A bad format that still passed through our tests ?